templates/corosync.conf.erb

```diff
@@ -41,7 +41,7 @@ totem {
 <% unless @threads.nil? -%>
   threads:                             <%= @threads %>
 <% end -%>
-<% if @unicast_addresses != 'UNSET' -%>
+<% unless @unicast_addresses.nil? -%>
```

Comment:
I think `if @unicast_addresses` can also work here?

Reply:
All occurrences of `unless @foo.nil?` were replaced with `if @foo`. This way `corosync.conf.erb` uses only one way to check variables.

Reply:
I replaced all occurences of `unless @foo.nil?` with `if @foo`. This way the template is more consistent.

Reply:
`unless @foo.nil?` is not equivalent to `if @foo`. The former matches defined variables, including `false`, while the latter does not match `false`.
The latest commit reverts some parts of `62302e03`.
@roidelapluie , any thought?

Reply:
Sorry, I wasn't aware of this. But none of the variables are boolean, so there is no need to match `false` anymore? Either a variable is definied or `undef`. When used as boolean `undef` is false.

Apart from commit message - which I will correct -, this change should be fine?